.travis.yml

```diff
@@ -5,37 +5,35 @@ dist: trusty
 env:
   global:
     - TRAVIS_NODE_VERSION="10"
+    - MODULE_NAME="webauthn-authenticator"
 
 matrix:
   include:
     - php: 7.1
-      env: DB=PGSQL RECIPE_VERSION=4.x-dev PHPCS_TEST=1 PHPUNIT_TEST=1
+      env: DB=PGSQL CORE_RELEASE=3.7 PHPCS_TEST=1 PHPUNIT_TEST=1
     - php: 7.2
-      env: DB=MYSQL RECIPE_VERSION=4.x-dev NPM_TEST=1
+      env: DB=MYSQL CORE_RELEASE=3.7 NPM_TEST=1
     - php: 7.3
-      env: DB=MYSQL RECIPE_VERSION=4.x-dev PHPUNIT_COVERAGE_TEST=1
+      env: DB=MYSQL CORE_RELEASE=3.7 PHPUNIT_COVERAGE_TEST=1
 
 before_script:
   - phpenv rehash
   - phpenv config-rm xdebug.ini
 
+  # Install composer
   - composer validate
-  - composer require --no-update silverstripe/recipe-cms:"$RECIPE_VERSION" silverstripe/recipe-testing:^1
-  - if [[ $DB == PGSQL ]]; then composer require --no-update silverstripe/postgresql:2.1.x-dev; fi
-  - composer install --prefer-source --no-interaction --no-progress --no-suggest --optimize-autoloader --verbose --profile
+  - git clone git://github.com/silverstripe/silverstripe-travis-support.git ~/travis-support
+  - php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss
+  - cd ~/builds/ss
 
   # Install NPM dependencies
-  - if [[ $NPM_TEST ]]; then nvm install $TRAVIS_NODE_VERSION && nvm use $TRAVIS_NODE_VERSION && npm install -g yarn && yarn install --network-concurrency 1 && (cd vendor/silverstripe/admin && yarn install --network-concurrency 1) && yarn run build; fi
+  - if [[ $NPM_TEST ]]; then (cd $MODULE_NAME && nvm install $TRAVIS_NODE_VERSION && nvm use $TRAVIS_NODE_VERSION && npm install -g yarn && yarn install --network-concurrency 1 && yarn run build); fi
 
 script:
-  - if [[ $PHPUNIT_TEST ]]; then vendor/bin/phpunit; fi
-  - if [[ $PHPUNIT_COVERAGE_TEST ]]; then phpdbg -qrr vendor/bin/phpunit --coverage-clover=coverage.xml; fi
-  - if [[ $PHPCS_TEST ]]; then vendor/bin/phpcs src/ tests/; fi
-  #- if [[ $NPM_TEST ]]; then git diff-files --quiet -w --relative=client; fi
-  #- if [[ $NPM_TEST ]]; then git diff --name-status --relative=client; fi
-  - if [[ $NPM_TEST ]]; then yarn run coverage; fi
-  - if [[ $NPM_TEST ]]; then yarn run lint; fi
-
-after_success:
-  - if [[ $PHPUNIT_COVERAGE_TEST ]]; then bash <(curl -s https://codecov.io/bash) -f coverage.xml; fi
-  - if [[ $NPM_TEST ]]; then bash <(curl -s https://codecov.io/bash) -F js; fi
+  - if [[ $PHPUNIT_TEST ]]; then vendor/bin/phpunit $MODULE_NAME; fi
+  - if [[ $PHPUNIT_COVERAGE_TEST ]]; then phpdbg -qrr vendor/bin/phpunit --coverage-clover=coverage.xml $MODULE_NAME; fi
+  - if [[ $PHPCS_TEST ]]; then vendor/bin/phpcs $MODULE_NAME/src/ $MODULE_NAME/tests/ *.php; fi
```

Comment:
The `*.php` here will catch everything in the project

Reply:
does it traverse directories by default? The old line was `src/ tests/ *.php`